images/linux/Ubuntu2004-Readme.md

```diff
@@ -196,23 +196,32 @@ to accomplish this.
 - nbgv 3.5.113+a908c6d9d2
 
 ### Databases
-- MongoDB 5.0.13 (apt source repository: https://repo.mongodb.org/apt/ubuntu)
 - sqlite3 3.31.1
 
+#### MongoDB
```

Comment:
Please don't change autogenerated doc file. You should update this script - https://github.com/actions/runner-images/blob/main/images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1 and functions B[uild-PostgreSqlSection](https://github.com/actions/runner-images/blob/145f9dff8f8a60a99a59ad3137383bd1fb7bea67/images/linux/scripts/SoftwareReport/SoftwareReport.Databases.psm1#L36), [Build-MySQLSection](https://github.com/actions/runner-images/blob/145f9dff8f8a60a99a59ad3137383bd1fb7bea67/images/linux/scripts/SoftwareReport/SoftwareReport.Databases.psm1#L51) and create a new one for MongoDb.

Reply:
Oh, that makes sense!  Would you like me to update this PR or make a new one?

Reply:
@knumat please reopen whenever you think you are ready